cg/apps/housekeeper/hk.py

```diff
@@ -573,3 +575,50 @@ def store_fastq_path_in_housekeeper(
             bundle_name=sample_internal_id,
             tag_names=[SequencingFileTag.FASTQ, flow_cell_id, sample_internal_id],
         )
+
+    def get_archive_entries(
+        self, archival_task_id: int = None, retrieval_task_id: int = None
+    ) -> list[Archive]:
```

Comment:
Should we throw and error if no `task_id` is provided?

Reply:
There is support for fetching all archives in that case, so I would say no @Vince-janv

Reply:
Sweet, can we mention that in a doc string?